MKOB/kobwindow.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, root, VERSION):
         self.root = root
         root.rowconfigure(0, weight=1)
         root.columnconfigure(0, weight=1)
-        root.title(VERSION)
+        root.title("MorseKOB " + VERSION)
```

Comment:
I stand by my suggestion that `VERSION` just be the version 'number', but seeing how many times this combination is used, it might be good to create a constant for it somewhere in the main mkob (`MKOB_VERSION_TEXT = "MorseKOB " + VERSION`) so you can use it consistently throughout.

Reply:
Let me know if you think this is worth a change. If you don't think so, I'm fine - maybe something to consider for the future (or you disagree - fine as well).

Reply:
Good suggestion. We might want to change the program name in the future. Best to have it in one place.

I made the change.